reduction_scripts/reduce_data.py

```diff
@@ -189,17 +187,18 @@ def run_overscan_sub(metadata, prev_suffix, curr_suffix):
     # repair bad pixels!
     def run_bpm_repair(metadata, prev_suffix, curr_suffix):
         full_obs_list = get_full_obs_list(metadata)
-        for fn in full_obs_list:
-            in_fn  = os.path.join(out_dir, '%s.p%s.fits' % (fn, prev_suffix))
-            out_fn = os.path.join(out_dir, '%s.p%s.fits' % (fn, curr_suffix))
-            if skip_done and os.path.isfile(out_fn):
+        for basename in full_obs_list:
+            input_filename =  f'{basename}.p{prev_suffix}.fits'
+            output_filename =  f'{basename}.p{curr_suffix}.fits'
+            input_filepath = os.path.join(out_dir, input_filename)
+            output_filepath = os.path.join(out_dir, output_filename)
+            if skip_done and os.path.isfile(output_filepath):
                 continue
-            print('Repairing %s bad pixels for %s' % (arm, in_fn.split('/')[-1]))
+            print(f'Repairing {arm} bad pixels for {input_filename}.')
```

Comment:
Just did a test run but saw this message:
```
Repairing red bad pixels for OBK-124192-WiFeS-Red--UT20230312T092550-8.p00.fits.
Calculating Global Superbias
Subtracting global superbias for OBK-124992-WiFeS-Red--UT20230829T063827-3.p01.fits
```
Might be worth removing the fullstop for consistency. Ideally all the messages would wrap the filename in single quotes, but it's probably not worth it.

Reply:
Other than that, all my results look the same as before.